src/status_im/contexts/onboarding/create_profile/view.cljs

```diff
@@ -0,0 +1,101 @@
+(ns status-im.contexts.onboarding.create-profile.view
+  (:require
+    [quo.core :as quo]
+    re-frame.db
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [status-im.common.events-helper :as events-helper]
+    [status-im.common.resources :as resources]
+    [status-im.config :as config]
+    [status-im.contexts.onboarding.create-profile.style :as style]
+    [status-im.contexts.onboarding.getting-started-doc.view :as getting-started-doc]
+    [utils.debounce :as debounce]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- navigate-to-create-profile-password
```

Comment:
don we have this in every control ?

Reply:
Thank you, @flexsurfer, for reviewing the PR.

Are you asking if we need the create-profile-password in both the create-profile and login flows? Yes, we do.

For the create profile flow, we are directly navigating to the create-profile-password screen here. However, for the login flow, navigation to the password screen occurs after the seed phrase is [validated](https://github.com/status-im/status-mobile/blob/0f036ca9c4fc88f8129cc6ce85bf90733f443f15/src/status_im/contexts/onboarding/events.cljs#L138).